src/ExpressionParser.php

```diff
@@ -506,10 +506,6 @@ public function parseSubscriptExpression($node)
             }
 
             if ($node instanceof NameExpression && null !== $this->parser->getImportedSymbol('template', $node->getAttribute('name'))) {
-                if (!$arg instanceof ConstantExpression) {
```

Comment:
`$arg` is always a `ConstantExpression`, initialized line 496.

This condition could be removed when code was moved in https://github.com/twigphp/Twig/commit/9f67d588c8a39a07ce0d78db56d9ef6a7d4b07f2.